src/librustdoc/html/render/mod.rs

```diff
@@ -706,7 +700,12 @@ fn short_item_info(
 
 // Render the list of items inside one of the sections "Trait Implementations",
 // "Auto Trait Implementations," "Blanket Trait Implementations" (on struct/enum pages).
-fn render_impls(cx: &Context<'_>, w: &mut Buffer, impls: &[&&Impl], containing_item: &clean::Item) {
+fn render_impls(
+    cx: &mut Context<'_>,
+    w: &mut Buffer,
+    impls: &[&&Impl],
```

Comment:
Lol this is some very strange code ... It's preexisting though, I'll open an issue.

Reply:
I mean, I only changed how `cx` was defined.